File: Android/sources/androidx/media3/exoplayer/source/ClippingMediaSource$IllegalClippingException.java
package androidx.media3.exoplayer.source;

import java.io.IOException;

public final class ClippingMediaSource$IllegalClippingException extends IOException {
    /* JADX INFO: super call moved to the top of the method (can break code semantics) */
    public ClippingMediaSource$IllegalClippingException(int i) {
        super("Illegal clipping: ".concat(i != 0 ? i != 1 ? i != 2 ? "unknown" : "start exceeds end" : "not seekable to start" : "invalid period count"));
    }
}

File: IOS/v2/iosx/media3/exoplayer/source/ClippingMediaSource$IllegalClippingException.swift
package iOSx.media3.exoplayer.source;.swift
import java.io.IOException;.swift
public final class ClippingMediaSource$IllegalClippingException extends IOException {.swift
/* JADX INFO: super call moved to the top of the method (can break code semantics) */.swift
public ClippingMediaSource$IllegalClippingException(int i) {.swift
super("Illegal clipping: ".concat(i = 1 ? i  "start exceeds end" : "not seekable to start" : "invalid period count"));.swift
}.swift
}.swift
